src/components/uielements/assets/assetData/AssetData.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import Coin from '../../coins/coin'
+import { CoinSize } from '../../coins/coin/types'
+import { AssetDataWrapper } from './AssetData.style'
+
+type Props = {
+  asset: string
+  assetValue?: number
+  price?: number
+  priceValue?: string
+  priceUnit: string
+  size?: CoinSize
+  className?: string
+}
+
+const AssetData: React.FC<Props> = (props: Props): JSX.Element => {
+  const { asset, priceValue, priceUnit, size = 'big', className = '', ...otherProps } = props
```

Comment:
Try to avoid to use `...otherProps`, it's just unsafe. _I know we did it in BEPSwap all over the place, but we should change now..._ I think there is a `lint` rule for this, which we should use to avoid it.